api/api.py

```diff
@@ -7,7 +7,60 @@ class Blob:
 
    To wrap a string as a blob, call ``blob(my_str)``"""
 
-def docker_build(ref: str, context: str, build_args: Dict[str, str] = {}, dockerfile: str = "Dockerfile", dockerfile_contents: Union[str, Blob] = "") -> None:
+class LiveUpdateStep:
+  """A step in the process of performing a LiveUpdate on an image's container.
```

Comment:
Is it worth linking all the LiveUpdateStep constructors from here, or let the docs handle that?

Reply:
UHHHH :-/ i'm still holding out hope that we can get the nesting to work. let me try that once we get the stuff merged, if not i'll go back and link stuff